src/test/java/com/mongodb/hibernate/jdbc/MongoConnectionTests.java

```diff
@@ -369,4 +374,44 @@ void testSQLExceptionThrownWhenMetaDataFetchingFailed() {
             assertThrows(SQLException.class, () -> mongoConnection.getMetaData());
         }
     }
+
+    @Nested
+    class ResultSetSupportTests {
+
+        private static final String EXAMPLE_MQL =
+                """
+                {
+                  find: "restaurants",
```

Comment:
Tests still pass even if the EXAMPLE_MQL is `{}`. Is that intended?

Reply:
yeah, it is not necessary. I got the example MQL from MongoDB doc website. I think a valid MQL might be more robust for testing, but in this case, I changed it to `{}` as you suggested.